### ISSUE TITLE
Strange Conflict Files

### ISSUE THREAD

**Issue by Creling** [vrtmrz/obsidian-livesync#279]
Hi,

There are some strange conflict files in my vault as the following snapshot shows:
![image](https://github.com/vrtmrz/obsidian-livesync/assets/43109504/e61f57a0-8858-4706-a720-baebb227ac7b)

The `Code01` in the file path is the name of another device. It seems that the configuration of `Code01` is synced to this device.

My problem is:
- Why do the conflicts appear here but not in the customized sync tab?
- How can I resolve these conflicts once and for all? I manually resolve them several times but they re-appear soon.

**Comment by vrtmrz** [comment]
Thank you for asking me! As you mentioned, the first piece of path (`Code01`) is the device name. This is the reason of device name is required to be unique among the devices.
Usually, these files would not be conflicted (The device should only update its information on the device). Therefore, the UI does not show the conflicts.
However, possibly the feature to handle issues about this might be useful. It will be improved. (It would be automatically resolved by a newer version).

**Comment by vrtmrz** [comment]
Since v0.19.20, Self-hosted LiveSync will resolve the conflicts of customized sync files automatically by their modified date.
Could you please check the behaviour, please?

**Comment by Creling** [comment]
> Since v0.19.20, Self-hosted LiveSync will resolve the conflicts of customized sync files automatically by their modified date. Could you please check the behaviour, please?

Yes, it works like a charm now. As you see, the conflict files are automatically resolved.

![image](https://github.com/vrtmrz/obsidian-livesync/assets/43109504/0dfe2a19-1ddf-4624-a589-e99e6b7bf0cd)
